modules/repository-azure/src/main/plugin-metadata/plugin-security.policy

```diff
@@ -14,4 +14,7 @@ grant {
   // Used by jackson bean deserialization
   permission java.lang.RuntimePermission "accessDeclaredMembers";
   permission java.lang.reflect.ReflectPermission "suppressAccessChecks";
+
+  // azure client reads the federated token file for workload identity authentication
+  permission java.io.FilePermission "${AZURE_FEDERATED_TOKEN_FILE}", "read";
```

Comment:
Should now be unnecessary following discussions elsewhere.

Reply:
Indeed though I have to rework the test setup to get the token into the config dir of the test cluster (and correctly point `AZURE_FEDERATED_TOKEN_FILE` to it)

Reply:
ah neat looks like:

```
.configFile("storage-azure/azure-federated-token", Resource.fromClasspath("azure-federated-token"))
        .systemProperty(
            "AZURE_FEDERATED_TOKEN_FILE",
            () -> "${ES_PATH_CONF}/storage-azure/azure-federated-token",
            s -> USE_FIXTURE && Strings.hasText(AZURE_TEST_CLIENT_ID) && Strings.hasText(AZURE_TEST_TENANT_ID)
        )
``` 

works